testdata/dnn/onnx/generate_onnx_models.py

```diff
@@ -263,3 +263,11 @@ def forward(self, x):
 model = Unsqueeze()
 model.eval()
 save_data_and_model("unsqueeze", input, model)
+
+input = Variable(torch.randn(1, 3, 8, 12, 16))
```

Comment:
Maybe we can do it smaller?

```
1*3*8*12*16*4 ~ 18.5KB
```
but, in exampe
```
input = Variable(torch.randn(1, 2, 3, 4, 5))
1*2*3*4*5*4 ~ 0.48KB (x38 smaller but also has vary dimensions).
```

Reply:
done